codalab/rest/interpret.py

```diff
@@ -316,14 +317,15 @@ def error_data(mode, message):
         if item is None:
             continue
         mode = item['mode']
-        data = item['interpreted']
-        properties = item['properties']
+        # TODO: remove these fields entirely, using fields unique to each mode instead.
```

Comment:
What's the migration plan?  Why keep these here if we plan to use the block-schema-specific fields...

Reply:
I'm going to update this PR to remove these entirely, they're in there currently to keep every other block type working while I test.